pandas/core/groupby/groupby.py

```diff
@@ -1835,7 +1835,7 @@ def rank(self, method='average', ascending=True, na_option='keep',
             The axis of the object over which to compute the rank.
 
         Returns
-        -----
+        -------
```

Comment:
Hmm OK - I suppose this is correct to do but was this showing up as a PR04 error?